### PR TITLE
Added short messages that explain why some functionality will not be available

### PR DESCRIPTION
**Pull request by asgrim**
Related to #7, this provides explanation as to why some functionality is not implemented

### CODE REVIEW

File: docs/compatibility.md [hunk 1]
```diff
@@ -25,14 +25,14 @@
 | getShortName | :heavy_check_mark: Yes |
 | getStartLine | todo |
 | getStaticProperties | todo |
-| getStaticPropertyValue | :x: No |
+| getStaticPropertyValue | :x: No - would require an instance |
```
Comment (Ocramius) [inline docs/compatibility.md]: You can create an accessor even without the instance of the object or internal reflection.

See http://ocramius.github.io/blog/accessing-private-php-class-members-without-reflection/

File: docs/compatibility.md [hunk 2]
```diff
@@ -42,13 +42,13 @@
...
 | isIterateable | todo |
-| isSubclassOf | :x: No |
+| isSubclassOf | todo |
 | isTrait | todo |
 | isUserDefined | todo |
 | newInstance | todo |
 | newInstanceArgs | todo |
 | newInstanceWithoutConstructor | todo |
-| setStaticPropertyValue | :x: No |
+| setStaticPropertyValue | :x: No - would require an instance |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 3]
```diff
@@ -124,9 +124,9 @@
 
 | Method | Supported |
 |--------|-----------|
-| getClosure | :x: No |
-| invoke | :x: No |
-| invokeArgs | :x: No |
+| getClosure | :x: No - would require actual compilation of the AST |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 4]
```diff
@@ -124,9 +124,9 @@
...
+| invoke | :x: No - would require loading of the function itself |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 5]
```diff
@@ -124,9 +124,9 @@
...
+| invokeArgs | :x: No - would require loading of the function itself |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 6]
```diff
@@ -138,14 +138,14 @@
 | getDocComment | todo |
 | getModifiers | todo |
 | getName | :heavy_check_mark: Yes |
-| getValue | :x: No |
+| getValue | :x: No - would require an instance of an object |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 7]
```diff
@@ -138,14 +138,14 @@
...
 | isDefault | todo |
 | isPrivate | :heavy_check_mark: Yes |
 | isProtected | :heavy_check_mark: Yes |
 | isPublic | :heavy_check_mark: Yes |
 | isStatic | :heavy_check_mark: Yes |
-| setAccessible | :x: No |
-| setValue | :x: No |
+| setAccessible | :x: No - would require an instance of an object |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 8]
```diff
@@ -138,14 +138,14 @@
...
+| setValue | :x: No - would require an instance of an object |
```
Comment (Ocramius) [inline docs/compatibility.md]: Same as above

File: docs/compatibility.md [hunk 9]
```diff
@@ -10,7 +10,7 @@
 | getDefaultProperties | todo |
 | getDocComment | todo |
 | getEndLine | todo |
-| getExtension | :x: No |
+| getExtension | :x: No - extensions are not supported |
```
Comment (Ocramius) [inline docs/compatibility.md]: Can rely on base reflection for this?

**Comment by asgrim** [comment]
@Ocramius I have created issues #14 and #15 to implement remaining compatibility issues, and linked them in the compatibility doc, as well as linking to #7 which is the "mostly" compatible task I'm working on

**Pull request merged**